MozillaFoundation/donate-wagtail
# updated upsell disclaimer to be a overridable block

## Description
Closes #1722 


As mentioned in #1722, the master template for the "card upsell view" did not include the block necessary for the template `thunderbird/card_upsell` to override the disclaimer text. This meant that the upsell page for thunderbird was using the Mozilla Foundations disclaimer text:


![Screenshot 2023-01-12 at 17-44-13 Give to Thunderbird](https://user-images.githubusercontent.com/18314510/212218776-9ed79ebf-4d4f-4510-a091-b1352dd5c9dc.png)



This PR updates the master template, and wraps the disclaimer text in the block needed so that the thunderbird template can override the text as expected:
![Screenshot 2023-01-18 at 10-54-24 Give to Thunderbird](https://user-images.githubusercontent.com/18314510/213269426-7db45516-57c8-4820-9cee-d28f8f5c606f.png)



## Steps to test

1. Check out this branch
2. Make a credit card donation greater than $20 with the test credit card information below
3. The upsell view disclaimer should have the following copy:
```
I authorize MZLA Technologies Corporation to automatically charge my card every month in the amount indicated above, 
starting next month on the same date as today, and continuing each month afterwards until I cancel. I understand that I must 
cancel at least 5 days before the next scheduled donation by submitting this form, and that I may only request a refund within 
15 days from the date of a donation.
```
4. If everything looks as expected, testing is complete! 


**Test Card Information:**
Card Number: 4111 1111 1111 1111
Exp: 11/23
CVV: 123
